package.json

```diff
@@ -40,10 +40,22 @@
   ],
   "moziot": {
     "api": {
-      "min": 1,
+      "min": 2,
       "max": 2
     },
+    "enabled": true,
```

Comment:
Leave this flag out of your `package.json`.

Reply:
Interesting! I added this line since I saw that thing-url-adapter had it

Reply:
We do that for thing-url-adapter, zigbee-adapter, and zwave-adapter since we pre-install them and there is no existing config database to determine whether or not those are enabled.